wildfly-standalone-centos/README.md

```diff
@@ -22,72 +22,77 @@
 <!-- /TOC -->
 
 ## Solution Overview
-WildFly 18 is the latest release in a series of JBoss open-source application server offerings. WildFly 18 is an exceptionally fast, lightweight and powerful implementation of the Jakarta Platform specifications. The state-of-the-art architecture built on the Modular Service Container enables services on-demand when your application requires them.
 
-This Azure quickstart template deploys a web applicaton deployed on WildFly 18.0.1.Final running on CentOS 8.
+This template creates a standalone node of WildFly on CentOS to a Standard F1 VM size in your Resource Group (RG) which includes a Public DNS, Private Virtual Network and security configuration. It is ideal for development and testing of enterprise Java applications on Azure. 
```

Comment:
I have added the paragraph as follows "This template creates a standalone node of WildFly on CentOS VM of Standard F1 size (users can select the size of the VM from the dropdown options) in your Resource Group (RG) which includes a Public DNS name, Virtual Network and Network Security Group. It is ideal for development and testing of enterprise Java applications on Azure." Please review and suggest